WooCommerce/WooCommerceTests/ViewRelated/Products/Actions Factory/ProductFormActionsFactoryTests.swift

```diff
@@ -792,6 +795,32 @@ final class ProductFormActionsFactoryTests: XCTestCase {
         assertEqual(expectedBottomSheetActions, factory.bottomSheetActions())
     }
 
+    func test_view_model_for_subscription_product_when_product_is_virtual() {
```

Comment:
❓ More of a question for my own learning: With this PR, we want to emphasize that a virtual subscription product should not have "Shipping: option in the bottom sheet action.

This unit test function is already correct, in the sense that shipping is not added in the "expectedBottomSheetsActions" part. But, there is nothing here that explicitly says we should not have "Shipping" action here. 

My question: is leaving things out here enough to communicate that requirement, or is it better to explicitly mention it (perhaps in a comment above the `let expectedBottomSheetActions: ` line below?

Reply:
Good observation, Hafiz! 🎉 

I agree that we can explicitly test for the "Shipping" section. Added a few more tests in https://github.com/woocommerce/woocommerce-ios/pull/11243/commits/24347e7d8461c6ed4d6b5351cf5dc69892482947

We can add more for variable subscription product type when we tackle https://github.com/woocommerce/woocommerce-ios/issues/11178